app/service/file_svc.py

```diff
@@ -169,12 +170,27 @@ def _read(self, filename):
         :param ldflags: A string of ldflags to use when building the go executable
         :param cflags: A string of CFLAGS to pass to the go compiler
         :param buildmode: GO compiler buildmode flag
+        :param build_dir: The path to build should take place in
         :return:
         """
-        os.system(
-            'cd %s && %s %s go build %s -o %s -ldflags=\'%s\' %s' % (build_dir, _go_vars(arch, platform), cflags,
-                                                                     buildmode, output, ldflags, src_fle)
-        )
+        env = copy.copy(os.environ)
+        env['GOARCH'] = arch
+        env['PLATFORM'] = platform
+        if cflags:
+            for cflag in cflags.split(' '):
+                name, value = cflag.split(':')
```

Comment:
I think this should be '=' instead of ':'
https://github.com/mitre/sandcat/blob/master/app/sand_svc.py#L39-L43

Reply:
oh yeah, good catch.  I think these should probably be passed in as a dict, but this function is used by plugins and I wanted to get this in sooner w/o a bunch of dependent PRs.